modules/fx/arrow-fx/src/test/kotlin/arrow/fx/IOTest.kt

```diff
@@ -681,31 +682,22 @@ class IOTest : UnitSpec() {
     }
 
     "can go from Either to IO directly when Left type is a Throwable" {
-
       val exception = RuntimeException()
       val left = Either.left(exception)
       val right = Either.right("rightValue")
 
-      left.toIO()
-        .attempt().unsafeRunSync() shouldBe Left(exception)
-
-      right.toIO()
-        .unsafeRunSync() shouldBe "rightValue"
+      left.toIO().unsafeRunSyncEither() shouldBe Left(exception)
```

Comment:
hmmmmmmmmmmmmmm

Reply:
Is this the one failing?

Reply:
Yes, and after #2003 everything works :)